ui/pages/import-tokens/index.scss

```diff
@@ -0,0 +1,85 @@
+
+.scrollable {
+  ::-webkit-scrollbar {
+    display: none; /* for Chrome, Safari, and Opera */
+  }
+}
+
+.banner {
+  margin-left: 20px;
+  margin-right: 20px;
+}
```

Comment:
Can this be `marginLeft ` and `marginRight` on a `<Box> component somewhere?

Reply:
I have tried using the props instead but i can't explain why it worked with className and not with marginLeft and marginRight.. You think it is okay to keep using the className?